BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -267,17 +277,17 @@ - (NSString *)sessionToken
 
 - (NSURLSessionDataTask *)signUpWithEmail:(NSString *)email username:(NSString *)username password:(NSString *)password completion:(SBBNetworkManagerCompletionBlock)completion
 {
-    return [_networkManager post:@"/api/v1/auth/signUp" headers:nil parameters:@{@"study":gSBBAppStudy, @"email":email, @"username":username, @"password":password, @"type":@"SignUp"} completion:completion];
+    return [_networkManager post:kSBBAuthSignUpAPI headers:nil parameters:@{@"study":gSBBAppStudy, @"email":email, @"username":username, @"password":password, @"type":@"SignUp"} completion:completion];
```

Comment:
Love the change to make the uri a constant. Is there any way to do something similar for the parameters? Some sort of mapping approach? It seems like a parameter name change in the future will be more painful and vulnerable to errors down the line. For example RestKit offers a convention based mapping to convert api parameters to object variables that I had liked in the past. I don't have the context to know if that has any relevance here and the typed out approach is something I see a lot, just wanted to highlight this. Definitely not a blocker.